jetty-core/jetty-util/src/test/java/org/eclipse/jetty/util/BufferUtilTest.java

```diff
@@ -57,6 +58,23 @@ public void afterEach()
         assertThat(FileSystemPool.INSTANCE.mounts(), empty());
     }
 
+    @Test
+    public void testSlice()
```

Comment:
Not necessary if we can use `ByteBuffer#slice(int,int)`

Reply:
`ByteBuffer.slice(int,int)` does not support the default values `BufferUtil.slice()` does.